frontend/store/index.ts

```diff
@@ -19,3 +19,4 @@ export * from './auth';
 export * from './items';
 export * from './playbackManager';
 export * from './taskManager';
+export * from './albumRuntimeBreakpoints';
```

Comment:
Where is this file in the PR?

Also, don't create an store just for this. For now, I think you can leave the values as const on top of the ``export default Vue.extend`` in the page content. Nothing is user-configurable at the moment, this kind of stuff should come up later and people that desire to make this kind of nitpicky changes can build the client themselves at the moment.

Adding configuration for every single thing right now is not feasible. Only major stuff that would live outside a convential config menu (like the history mode for Vue router or the default servers) are welcome in JSON config files (as they should be) or env vars (as they're now).

Reply:
I must have failed to include it in my commit? I thought I did. I'll change it to your recommendation.